lib/stripe/subscriptions/coupon.ex

```diff
@@ -19,18 +19,18 @@ defmodule Stripe.Coupon do
   @type t :: %__MODULE__{
                id: Stripe.id,
                object: String.t,
-               amount_off: pos_integer,
-               created: Stripe.timestamp,
-               currency: String.t,
-               duration: :forever | :once | :repeating,
+               amount_off: pos_integer | nil,
+               created: pos_integer,
```

Comment:
Timestamps should be a `Stripe.timestamp()` with `| nil` if necessary (or any other additional constraints).

Reply:
The created was listed as an integer in the yaml and described as the number of seconds from the epoch.  Shouldn't those be kept as integers, or are we saying that we convert epoch values to timestamps?